src/com/home_work/spring_boot/ui/UIRepresentation.java

```diff
@@ -0,0 +1,93 @@
+package com.home_work.spring_boot.ui;
+
+import com.home_work.spring_boot.entity.Letter;
+import com.home_work.spring_boot.services.MailService;
+import com.home_work.spring_boot.services.ValidationService;
+import com.home_work.spring_boot.threads.MailSenderThread;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.swing.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+@Component
+public class UIRepresentation {
+
+    private ValidationService validationService;
+    private MailService mailService;
+    private Letter letter = new Letter();
+    ExecutorService executorService = Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors());
```

Comment:
this should be created in separate configuration. do not initialize your dependencies, let spring do it